nuxt/components/edit/EditDraftLyrics.vue

```diff
@@ -0,0 +1,252 @@
+<template>
+  <v-dialog
+    v-model="dialog"
+    persistent
+    fullscreen
+    no-click-animation
+    hide-overlay
+  >
+    <template #activator="{ on }">
+      <v-btn
+        icon
+        v-on="on"
+      >
+        <v-icon>lyrics</v-icon>
+      </v-btn>
+    </template>
+    <v-card :loading="loading">
+      <v-app-bar
+        fixed
+        elevate-on-scroll
+      >
+        <v-btn
+          icon
+          @click="close"
+        >
+          <v-icon>close</v-icon>
+        </v-btn>
+        <v-toolbar-title>Draft Lyrics for {{ track.title }}</v-toolbar-title>
```

Comment:
Let's just use `Edit Write-Up` as the title

Reply:
Done